cogeo_mosaic/backends/sqlite.py

```diff
@@ -132,7 +132,13 @@ def write(self, overwrite: bool = False):
                         maxzoom INTEGER NOT NULL,
                         quadkey_zoom INTEGER,
                         bounds JSON NOT NULL,
-                        center JSON
+                        center JSON,
+                        tilematrixset JSON,
+                        asset_type TEXT,
+                        asset_prefix TEXT,
+                        data_type TEXT,
+                        colormap JSON,
+                        layers JSON
```

Comment:
Note: This is kinda tricky but the new cogeo-mosaic version won't be able to work with sqlite mosaicJSON created with previous version. 

maybe we could add a way to `update` the metadata_table (add column)